notebooks/233-blip-visual-language-processing/233-blip-visual-language-processing.ipynb

```diff
@@ -150,7 +150,7 @@
     }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        ov.save_model(ov_vision_model, str(VISION_MODEL_OV))
</pre><p>str not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>